.travis.yml

```diff
@@ -5,10 +5,9 @@ os:
 language: node_js
 node_js:
 - node
-- '9'
-- '8'
-- '7'
-- '6'
+- '9' # EOL: 2018-06-30
```

Comment:
9 looks past EOL.
Is 10 supported yet?

Reply:
10 is active LTS, 11 is current release. When we specify `node` in this list, it maps to 11

Reply:
Remove 9?

Reply:
Sure, add 10 also?